ui/hooks/useGasFeeEstimates.js

```diff
@@ -25,14 +32,44 @@ import { useSafeGasEstimatePolling } from './useSafeGasEstimatePolling';
  * GasFeeController that it is done requiring new gas estimates. Also checks
  * the returned gas estimate for validity on the current network.
  *
+ * @param _networkClientId - The optional network client ID to get gas fee estimates for. Defaults to the currently selected network.
  * @returns {GasEstimates} GasEstimates object
  */
-export function useGasFeeEstimates() {
-  const gasEstimateType = useSelector(getGasEstimateType);
-  const gasFeeEstimates = useSelector(getGasFeeEstimates, isEqual);
-  const isGasEstimatesLoading = useSelector(getIsGasEstimatesLoading);
-  const isNetworkBusy = useSelector(getIsNetworkBusy);
-  useSafeGasEstimatePolling();
+export function useGasFeeEstimates(_networkClientId) {
+  const selectedNetworkClientId = useSelector(getSelectedNetworkClientId);
+  const networkClientId = _networkClientId ?? selectedNetworkClientId;
```

Comment:
should we throw an error if somehow this isn't defined?

Reply:
i think we just let it blow up downstream. selectedNetworkClientId coming back undefined would signal something worse is messed up